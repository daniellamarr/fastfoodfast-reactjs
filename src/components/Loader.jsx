import React, { Component } from 'react';

const loader = require('../../public/images/loader.gif');

/**
 * Loader
 */
export default class Loader extends Component {
  /**
   * @returns {JSX} jsx
   */
  render() {
    return (
      <div id="loader">
        <center>
          <img src={loader} alt="loading..." width="80px" />
        </center>
      </div>
    );
  }
}
