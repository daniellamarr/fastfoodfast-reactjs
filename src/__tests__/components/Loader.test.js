/* eslint-disable react/react-in-jsx-scope */
/* eslint-disable no-undef */
import { shallow } from 'enzyme';
import Loader from '../../components/Loader.jsx';

let wrapper;

beforeEach(() => {
  wrapper = shallow(<Loader />);
});

describe('Loader Component', () => {
  it('should match snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
