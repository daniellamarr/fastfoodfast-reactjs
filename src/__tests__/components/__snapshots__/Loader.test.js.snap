// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader Component should match snapshot 1`] = `
<div
  id="loader"
>
  <center>
    <img
      alt="loading..."
      src="test-file-stub"
      width="80px"
    />
  </center>
</div>
`;
